Sources/xcodeproj/Objects/Project/PBXProject.swift

```diff
@@ -191,6 +191,44 @@ public final class PBXProject: PBXObject {
 
         return reference
     }
+    
+    
+    /// Adds a local swift package
+    ///
+    /// - Parameter path: Relative path to the swift package
+    /// - Parameter productName: The product to depend on
+    /// - Parameter target: Target to link package product to
+    /// - Parameter addFileReference: Include a file reference to the package (defaults to main group)
+    public func addLocalSwiftPackage(path: String,
+                                     productName: String,
+                                     target: PBXTarget? = nil,
```

Comment:
I think it's safe here to assume that the developer wants to add the package to an existing target, and therefore I'd make this a non-optional attribute. If developers are only interested in adding this to the project, they can implement it themselves.

Reply:
Good point 👍

Reply:
Now, is not that the same for `addSwiftPackage`? If so, probably would make sense to change it to that, too.

Reply:
Yes, I'd change it too